test/downstream/delaydiffeq.jl

```diff
@@ -31,3 +31,20 @@ DDEProblemLibrary.importddeproblems()
         @test sol[1, :] ≈ sol_scalar.u
     end
 end
+
+τ = 1.0
+function latka_volterra!(du,u,h,p,t)
```

Comment:
Is there a specific reason for the name `latka_volterra!` instead of `lotka_volterra!`?

Reply:
Nope, copy pasted from a user code "copy pasted" from my live coding workshop where I made a mistake in the spelling 🤷 Indeed let's fix this.